smoketest.bash

```diff
@@ -66,16 +67,10 @@ while getopts "hs:prGtAOVXcbnk" opt; do
             FILES+=(
                 "${DIR}/compose/sample-apps.yml"
                 "${DIR}/compose/sample-apps_https.yml")
```

Comment:
I guess you could remove the `FILES+=` part here and just set the variable. Then after the flag processing is done, you can use `INCLUDE_SAMPLE_APPS` and `SAMPLE_APPS_USE_TLS` to decide which files to add to the array. This is already working as intended but maybe doing this would be a nice cleanup/refactoring.